### DISCUSSION TITLE
docker login does not work via GUI

### DISCUSSION THREAD

**Discussion by p10tyr** [louislam/dockge#492] [Ask for Help]
### ⚠️ Please verify that this bug has NOT been reported before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### Description

When I use the GUI bash terminal to do docker login and enter my username. It says `command is not allowed` ?

I log in with SSH and do docker login, authenticated OK.

Press start on stack

### 👟 Reproduction steps

- Add a private image in stack
- docker login in GUI does not work
- docker login in SSH - Authenticated OK
- start stack
- error

`Error response from daemon: pull access denied for private-reop/cool-image, repository does not exist or may require 'docker login': denied: requested access to the resource is denied`

pull in ssh - pulled OK

### 👀 Expected behavior

image is pulled in GUI after login in SSH

### 😓 Actual Behavior

`Error response from daemon: pull access denied for private-reop/cool-image, repository does not exist or may require 'docker login': denied: requested access to the resource is denied`

### Dockge Version

latest

### 💻 Operating System and Arch

debian

### 🌐 Browser

edge

### 🐋 Docker Version

latest

### 📝 Relevant log output

```shell
`Error response from daemon: pull access denied for private-reop/cool-image, repository does not exist or may require 'docker login': denied: requested access to the resource is denied`
```

**Comment by p10tyr** [comment]
Instead of trying to log in via dockge console, do this

1)  `sudo docker login` - to create a .docker file for docker in /root
1) uncomment line `dockage/compose.yml`
1) `sudo docker compose up -d` dockage compose file

**Comment by Nick390** [comment]
I try this
```
root@localhost:~# sudo docker login
Authenticating with existing credentials...
WARNING! Your password will be stored unencrypted in /root/.docker/config.json.
Configure a credential helper to remove this warning. See
https://docs.docker.com/engine/reference/commandline/login/#credentials-store
```
Where I can find `dockage/compose.yml` ?

**Reply by p10tyr** [reply]
The compose.yml file is the one you used to run dockge the first time. Its inside that file 😄
